cfpb/cfgov-refresh
# Fix for nav urls

Fix for nav urls

## Changes

- Modifying nav urls to prepend a slash. This fixes an issue with navigating relative paths. 

## Testing

1. Visit `http://localhost:8000/practitioner-resources/financial-coaching/` via the nav.
2. Visit http://localhost:8000/practitioner-resources/financial-coaching/ and verify the page loads as expected.

## Screenshots


## Notes

-

## Todos

-

## Checklist


## Testing checklist

### Browsers


### Accessibility


### Other
